docs/guide/insight.md

```diff
@@ -0,0 +1,34 @@
+title: Insight Explorer
+description: Allows users to fetch information about the state of the blockchain from a trusted Insight server.
```

Comment:
This may be better: "Provides an interface to fetch information about the state of the blockchain from a trusted Insight server."